### PR TITLE
TER-273 YAML format for saving list of platforms

### PR DESCRIPTION
**Pull request by 26christy**
platform.yaml template to be added to the farm repo

### CODE REVIEW

File: examples/farm/platform/platform.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) CloudCover
+# SPDX-License-Identifier: Apache-2.0
+
+platforms:
+  - name: postgres
+    repo_url: https://github.com/your/repo
+    repo_directory: examples/farm/platform/
+    ref_label: v1.0 # The label can be tag/branch/commit
+    label_type: 2 # 1=branch, 2=tag, 3=commit
```
Comment (kanaksinghal) [inline examples/farm/platform/platform.yaml:9]:
```suggestion
    label_type: tag # one of: 'branch', 'tag', 'commit'
```

this yaml format should be user friendly, so we should let users put the type in string. internally we should change it to enum for efficiency if needed.

File: examples/farm/platform/platform.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) CloudCover
+# SPDX-License-Identifier: Apache-2.0
+
+platforms:
+  - name: postgres
```
Comment (kanaksinghal) [inline examples/farm/platform/platform.yaml:5]: We need to think more about what this `name` attribute is here, if we want to use it for display purpose on the UI, then it should be calls `title` & `description` for consistency.
I can't think of any other usage of this at this time... 🤔?

also, postgres, redis would be bad example. it should be `AWS EKS`, `AWS Serverless`, `GCP GKE`, `GCP Serverless`, etc.

**Comment by kanaksinghal** [comment]
we'll need parser for the yaml format as well, do want to add that in this ticket or new?

**Comment by 26christy** [comment]
> we'll need parser for the yaml format as well, do want to add that in this ticket or new?

@kanaksinghal i have written the parser in TER-277 along with the harvest command

File: examples/farm/platform/platform.yaml (hunk 2 above)
Comment (26christy) [reply examples/farm/platform/platform.yaml:5]: Ya got it. That make sense to achieve consistency

**Review by kanaksinghal — APPROVED** [review]

**Pull request merged**